src/explore-education-statistics-admin/src/pages/publication/PublicationReleasesPage.tsx

```diff
@@ -1,53 +1,17 @@
 import ButtonLink from '@admin/components/ButtonLink';
-import PublicationDraftReleases from '@admin/pages/publication/components/PublicationDraftReleases';
 import PublicationPublishedReleases from '@admin/pages/publication/components/PublicationPublishedReleases';
-import PublicationScheduledReleases from '@admin/pages/publication/components/PublicationScheduledReleases';
+import PublicationUnpublishedReleases from '@admin/pages/publication/components/PublicationUnpublishedReleases';
 import usePublicationContext from '@admin/pages/publication/contexts/PublicationContext';
 import { PublicationRouteParams } from '@admin/routes/publicationRoutes';
 import { releaseCreateRoute } from '@admin/routes/routes';
-import publicationService from '@admin/services/publicationService';
-import LoadingSpinner from '@common/components/LoadingSpinner';
-import WarningMessage from '@common/components/WarningMessage';
-import useAsyncHandledRetry from '@common/hooks/useAsyncHandledRetry';
-import React, { useMemo } from 'react';
+import noop from 'lodash/noop';
+import React, { useRef } from 'react';
 import { generatePath } from 'react-router';
 
 const PublicationReleasesPage = () => {
-  const { publicationId } = usePublicationContext();
+  const { publicationId, publication } = usePublicationContext();
 
-  // To ensure the releases are up to date when you switch to this tab
-  // we're re-fetching the publication here instead of using it from the context.
-  // This is not ideal and will be replaced by a call to just get the publication
-  // releases when an endpoint for this is ready - EES-3554 .
-  const {
-    value: publication,
-    isLoading: loadingPublication,
-    retry: reloadPublication,
-  } = useAsyncHandledRetry(() =>
-    publicationService.getMyPublication(publicationId),
-  );
-
-  const draftReleases = useMemo(() => {
-    return (
-      publication?.releases.filter(
-        release =>
-          release.approvalStatus === 'Draft' ||
-          release.approvalStatus === 'HigherLevelReview',
-      ) ?? []
-    );
-  }, [publication?.releases]);
-
-  const publishedReleases = useMemo(() => {
-    return publication?.releases.filter(release => release.live) ?? [];
-  }, [publication?.releases]);
-
-  const scheduledReleases = useMemo(() => {
-    return (
-      publication?.releases.filter(
-        release => !release.live && release.approvalStatus === 'Approved',
-      ) ?? []
-    );
-  }, [publication?.releases]);
+  const publishedReleasesRefetchRef = useRef<() => void>(noop);
```

Comment:
I doesn't look like the releases aren't being refetched when you switch tabs any more.

Reply:
Good point. It's because I'd set the React Query `staleTime` option to infinity unnecessarily. Just revert this back to using the defaults!